src/serverConfig-example.ts

```diff
@@ -3,23 +3,33 @@
 // Best is to Keep them as a reference
 // Just duplicate the file and name it serverConfig.ts to adjust to your likings
 
+
+// you can derive the following options by looking at the /index.html file of your jitsi-meet-web instance (i.e. the server, where you can create new sessions)
+
 export const connectionOptions = {
   hosts: {
-     domain: 'meet.jit.si',
-     muc: 'conference.meet.jit.si', 
-     focus: 'focus.meet.jit.si',
-  }, 
-  externalConnectUrl: 'https://meet.jit.si/http-pre-bind', 
-  // enableP2P: true, 
-  // p2p: { 
-  //    enabled: true, 
-  //    preferH264: true, 
-  //    disableH264: true, 
+    // curl -s https://$JITSI_MEET_WEB_DOMAIN/index.html | grep config.hosts.domain
+    domain: 'meet.jit.si',
+    // curl -s https://$JITSI_MEET_WEB_DOMAIN/index.html | grep config.hosts.muc
+    muc: 'conference.meet.jit.si',
+    // curl -s https://$JITSI_MEET_WEB_DOMAIN/index.html | grep config.hosts.anonymousdomain
+    anonymousdomain: 'guest.meet.jit.si',
+    // curl -s https://$JITSI_MEET_WEB_DOMAIN/index.html | grep config.hosts.authdomain
+    authdomain: 'meet.jit.si',
+  },
+  // propbably https://$JITSI_MEET_WEB_DOMAIN/http-pre-bind
+  externalConnectUrl: 'https://meet.jit.si/http-pre-bind',
+  // enableP2P: true,
+  // p2p: {
+  //    enabled: true,
+  //    preferH264: true,
+  //    disableH264: true,
   //    useStunTurn: true,
-  // }, 
+  // },
   // useStunTurn: true,
-  bosh: `https://meet.jit.si/http-bind?room=chatmosphere1234`, // ! if you make your own please omit the "?room=chatmosphere1234" part
+  //bosh: `https://meet.jit.si/http-bind?room=chatmosphere1234`, // ! if you make your own please omit the "?room=chatmosphere1234" part
```

Comment:
Why is this now commented out (which is a change from the default)?

Reply:
as above: not sure what it is needed for / whether it is needed.

Reply:
It's the URL used to contact the underlying Prosody XMPP server via [BOSH](https://xmpp.org/extensions/xep-0124.html). You don't explicitly need it as `/http-bind` is the default in the [library wrapper](https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/xmpp/xmpp.js) which gets passed the options object. Also it looks like it was renamed to `serviceURL` so the value in this config files doesn't take effect but I'm not sure on that.